appserver/extras/payara-micro/payara-micro-core/src/main/java/fish/payara/micro/PayaraMicro.java

```diff
@@ -265,6 +275,30 @@ public void setUserLogFile(String fileName) {
     }
     
     /**
+     * Set user defined file directory for the access log
+     *
+     * @param filePath
+     */
+    public void setAccessLogDir(String filePath) {
+        if (!filePath.endsWith("/")) {
+
```

Comment:
If value does not end in "/", the value is ignored and access log file is not created. We could treat access log argument in the same way as logFile argument - if "/" is not at the end, it specifies path to a file instead of directory. Even better, I would fix both access log and userLogFile to add a check whether the passed argument is a file or directory, in case "/" is not at the end. Otherwise people might create a directory beforehand and pass it as argument without a slash, and this would cause an error - Payara Micro would expect it to be a file, but it is a directory.